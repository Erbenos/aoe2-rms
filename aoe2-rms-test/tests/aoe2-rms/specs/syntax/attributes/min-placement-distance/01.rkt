#lang aoe2-rms
; Create three deserts that have their origins at least 25 tiles from the other deserts.

<LAND-GENERATION>
(create-land
 (terrain-type 'DESERT)
 (land-percent 1)
 (min-placement-distance 25))

(create-land
 (terrain-type 'DESERT)
 (land-percent 1)
 (min-placement-distance 25))

(create-land
 (terrain-type 'DESERT)
 (land-percent 1)
 (min-placement-distance 25))
