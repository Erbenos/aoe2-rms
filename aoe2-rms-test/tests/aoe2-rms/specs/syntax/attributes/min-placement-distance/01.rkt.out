<LAND_GENERATION>
create_land {
  terrain_type DESERT
  land_percent 1
  min_placement_distance 25
}
create_land {
  terrain_type DESERT
  land_percent 1
  min_placement_distance 25
}
create_land {
  terrain_type DESERT
  land_percent 1
  min_placement_distance 25
}
